src/databricks/labs/ucx/runtime.py

```diff
@@ -67,6 +69,37 @@ def inventorize_mounts(cfg: MigrationConfig):
     mounts.inventorize_mounts()
 
 
+@task("assessment", depends_on=[inventorize_mounts, crawl_tables])
+def inventorize_external_locations(cfg: MigrationConfig):
+    """In this part of the assessment, we're going to look up the location of all the tables. Locations that use
+    Mount Points will be looked up. We will then find out the External Locations required for a proper migration and
+    save these to a Table.
```

Comment:
can you expand a bit more on documenting the logic for coming up with external locations? this is a user-facing documentation